servers_v6.json

```diff
@@ -17,7 +17,7 @@
   },
   {
     "name": "C.A.M.S.",
-    "address": ["routerchain.ddns.net", "nikochio.ddns.net"]
+    "address": ["routerchain.ddns.net", "nikochio.ddns.net", "mindustry.ru"]
```

Comment:
```suggestion
    "address": ["routerchain.ddns.net", "nikochio.ddns.net"]
```

Reply:
Why Merge mindustry.ru on C.A.M.S list

Reply:
.ru was added on cams by router which was taken as "cams is p2w now"  and therefore griefing factions took action

Reply:
ok